output_from_kcl.py

```diff
@@ -0,0 +1,161 @@
+import asyncio
+import os
+import re
+import tomllib
+from concurrent.futures import ProcessPoolExecutor
+from io import BytesIO
+from pathlib import Path
+from typing import Container
+
+import kcl
+import requests
+from kcl import UnitLength
+from PIL import Image
+
+RETRIES = 5
+
+# Map strings to kcl units
+UNIT_MAP = {
+    'in': kcl.UnitLength.In,
+    'mm': kcl.UnitLength.Mm,
+    'ft': kcl.UnitLength.Ft,
+    'm': kcl.UnitLength.M,
+    'cm': kcl.UnitLength.Cm,
+    'yd': kcl.UnitLength.Yd,
+}
+
+
+def export_step(code: str, save_path: Path, unit_length: UnitLength = kcl.UnitLength.Mm) -> bool:
+    try:
+        export_response = asyncio.run(
+            kcl.execute_and_export(code, unit_length, kcl.FileExportFormat.Step)
+        )
+
+        stl_path = save_path.with_suffix(".step")
+
+        with open(stl_path, "wb") as out:
+            out.write(bytes(export_response[0].contents))
+
+        return True
+    except Exception as e:
+        print(e)
+        return False
+
+
+def find_files(
+        path: str | Path, valid_suffixes: Container[str], name_pattern: str | None = None
```

Comment:
Nit: Could just use `list` for the type hint here instead of importing and using `Container` since you pass in `['.kcl']` later on. Any iterable works though.

```suggestion
        path: str | Path, valid_suffixes: list[str], name_pattern: str | None = None
```